.github/workflows/ci-actions.yml

```diff
@@ -501,11 +501,13 @@ jobs:
             test-modules: >
               kubernetes-client
           - category: Misc4
-            timeout: 30
+            timeout: 45
             test-modules: >
               smallrye-graphql
               picocli-native
               gradle
+              micrometer-mp-metrics
+              micrometer-prometheus
```

Comment:
Do we have `micrometer-jmx` somewhere in the CI? Isn't it missing?

Reply:
Does it have native tests? If not, there's no need.

Reply:
Oh. You're right.

Reply:
I didn't set up all of them. There is a native test for Micrometer + JMX, but I didn't want to extend the build time too much. If/when we have incremental builds, I would make a separate bucket for micrometer and go to town.